### DISCUSSION TITLE
Replace Save or Publish Buttons

### DISCUSSION THREAD

**Discussion by zubricks** [payloadcms/payload#1324] [Roadmap]
We should make it possible to extend Payload so that users can slot in custom save or publish buttons and tie them to alternate actions as needed —. e.g. _Request Review_ or similar.

Marking as planned and logging here for visibility.

Related:
https://github.com/payloadcms/payload/discussions/1009

**Comment by jacobsfletch** [comment]
Related: https://github.com/payloadcms/payload/discussions/1996

**Comment by pcbal** [comment]


**Reply by ssyberg** [reply]
💪🏼

**Comment by JarrodMFlesch** [comment]
👀 [2613](https://github.com/payloadcms/payload/pull/2613)

**Comment by robbie-hunt** [comment]
Just want to say that I've started using Payload specifically to build a content management system for a news organisation where authors can log in, write and save their stories as drafts, and then set the status from Draft to Pending Review, at which point the editor attached to the story can view the story and change status to Draft or Publish, or keep it at Pending Review.

I have extremely limited JS skills and thought that doing this 'Pending Review' feature wouldn't be much more hassle than a lot of the configuration for Collections and other extensible parts of Payload already is. I haven't attempted it yet but I'm glad to see it's on the roadmap, it's an essential feature for me in using Payload, if I can't make it work I won't be able to use Payload to create this 'story pipeline' for this news organisation

**Reply by ssyberg** [reply]
JFYI we ended up basically replacing Payloads native publishing flow wholesale to support this (and some related features), I'm not sure I'd suggest attempting that with limited JS skills (and it requires a pretty thorough understanding of Payload itself). Your best option for the moment would probably be to rely on `draft` state and keep the rest of the workflow out of Payload.

**Reply by robbie-hunt** [reply]
@ssyberg thanks very much for the information, I'll try to rely on the Draft status for now. Excited for this feature
